R/loopRuns.R

```diff
@@ -14,17 +16,19 @@
 #' loopRuns(dir())
 #' }
 #'
-loopRuns <- function(mydir, user = NULL) {
```

Comment:
added these options such that `-a` and `-c` output still look the same as before